### ISSUE TITLE
[SelectDialog]: A11y Item names pronounced multiple times when navigating with TalkBack

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11470]
### Bug Description

When using the `SelectDialog` component on a mobile device with TalkBack enabled, an issue occurs while navigating through list items using swipe gestures. 

Specifically:

When swiping to move focus to a list item, the item's name is pronounced twice by TalkBack.
If the `accessibleName` property is added to the` ListItemStandard` component, the item's name begins to be pronounced three times instead of twice.

### Affected Component

SelectDialog

### Expected Behaviour

When navigating through list items with TalkBack enabled, the item's name should be pronounced only once.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/modals-popovers-selectdialog--multi-select

### Steps to Reproduce

1. Open example on mobile device https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/modals-popovers-selectdialog--multi-select
2. Enable TalkBack on your mobile device.
3. Use swipe gestures to navigate through the list items.
4. Observe the pronunciation behavior.

### Priority

None

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Operating System

Android

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Dear @AlekseiSch, 

In order to make sure that the issue you reported in not specific to Android as an OS and Talkback as its native screenreader, you need to make sure that the issue is reproducible in another environment. In this case the closes will probably be VoiceOver and IOS. 

Please re-test on a  different environment and report back if you think there is a general issue at hand. 
More on screen reader support can be found at: 
https://sap.github.io/ui5-webcomponents/docs/advanced/accessibility/#screen-reader-support

Kind regards, 

Georgi

SAP Development Support

**Comment by AlekseiSch** [comment]
On IPhone with VoiceOver only the checkbox is announced, and information is only announced when quick nav is enabled.
On Mac counters (2 of 40) are announced twice.

![Image](https://github.com/user-attachments/assets/9af21448-7f7f-457b-8aa3-507669050aec)

**Comment by Valkanister** [comment]
Hello @AlekseiSch, 

I tested with JAWS and NVDA and in both cases there is no double announcement. The checkbox itself has tabindex="-1" which means it is not part of the TAB chain. How it gets focused with TalkBack or Voiceover seems to be specific to those two screen readers. 
Currently we see no issue in the accessibility of the control. 

Kind regards, 

Georgi

SAP Development Support
